gatsby/content/docs/e2ee_cross_signing.mdx

```diff
@@ -0,0 +1,1198 @@
+---
+layout: post
+title: Implementing more advanced e2ee features, such as cross-signing
+section: Encryption
+---
+
+PLEASE NOTE THAT THIS GUIDE IS STILL A WIP!
+
+# A guide to more advanced e2ee features of matrix
```

Comment:
The title from the front matter above gets inserted as an h1, used as the title and the slug. This is for consistency. You don't need to have a second title. Pick one to use as the title, and if you prefer you could keep the second one just as a subhead (can be regular text)